examples/crc-create-myadmin.sh

```diff
@@ -0,0 +1,8 @@
+oc create user myadmin
```

Comment:
Let's move this to `docs/downstream` as well

Reply:
In that case I'd suggest putting this in a markdown, it would be weird having a script in a docs folder :)

Reply:
I'd rather put it to `examples/`. But can create `examples/upstream/crc` or `examples/crc` WDTY?

It probably does not matter much at this point and we can figure it up as we add more content anyway.